inspectit-gepard-agent/src/main/java/rocks/inspectit/gepard/agent/internal/identity/IdentityManager.java

```diff
@@ -0,0 +1,76 @@
+/* (C) 2024 */
+package rocks.inspectit.gepard.agent.internal.identity;
+
+import java.lang.management.ManagementFactory;
+import java.lang.management.RuntimeMXBean;
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+import java.nio.charset.StandardCharsets;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import rocks.inspectit.gepard.agent.internal.identity.model.IdentityInfo;
+
+public class IdentityManager {
+
+  private static final Logger log = LoggerFactory.getLogger(IdentityManager.class);
+
+  private final IdentityInfo identityInfo;
+
+  private IdentityManager() {
+    RuntimeMXBean runtime = ManagementFactory.getRuntimeMXBean();
+    long pid = runtime.getPid();
+    String localHostName = getLocalHostname();
```

Comment:
`runtime.getName()` would return the name of the JVM in the format `<pid>@<hostname>`
 
 Maybe we can use this one? What do you think?

Reply:
Will check it.